src/System.Memory/src/System.Memory.builds

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Memory.csproj" Condition="'$(OS)'=='Windows_NT'" />
```

Comment:
S.R.CS.Unsafe is built on Windows only. I suspect that it is because of ilproj support was not available in  Unix buildtools at the time it was introduced. (It is no longer the case so it should be possible to remove this condition even for S.R.CS.Unsafe.)

System.Memory does not need ilasm to build. It uses S.R.CS.Unsafe from package so I do not think there is good reason for having this condition here.